packages/nimble-components/src/wafer-map/modules/event-coordinator.ts

```diff
@@ -12,25 +13,35 @@ export interface EventCoordinatorCallbacks {
  */
 export class EventCoordinator {
     private readonly zoomHandler;
-    private readonly hoverHandler;
+    private readonly stableHoverHandler;
+    private readonly expHoverHandler;
```

Comment:
Even though this is a temporary name, I still suggest ditching the "exp" short-hand for "experimental".